src/pages/NotFoundPage.js

```diff
@@ -0,0 +1,39 @@
+import React from 'react'
+import styled from 'styled-components'
+import { Link } from 'react-router-dom'
+import home from '../images/home.svg'
+
+export default function NotFound() {
+  return (
+    <>
+      <StyledMessage>404 - Page not found</StyledMessage>
+      <Link to="/">
```

Comment:
Noch eine andere Idee, aber ich kenne mich mit dem Router zu wenig aus als dass ich mit Bestimmtheit sagen kann ob das klappt - kannst du eine Link Komponente bauen, in der du als props die Route und src für den Icon mit durchgibst? Dann brauchst du nur den Link aufzurufen und hast auch die Styled components zum Icon und dem Div nur einmal. Stelle mir das etwa so vor: Der aufruf hier: 
```suggestion
     <StyledRouterLink icon={home} route={"/"} />
```
und in einer eigenen Komponente:
```
export default function StyledRouterLink({ icon, route }) {
  return (
    <Link to={route}>
    <StyledDiv>
      <StyledIcon src={icon} />
    </StyledDiv>
</Link>
  )
}
```

Reply:
Hallo Svenja,
lieben Dank für deine Zeit und dein ausführliches Feedback! Das sind auf jeden Fall super wertvolle Punkte! :-)

Reply:
Werde ich definitiv angehen.